libs/anglify/src/lib/modules/checkbox/checkbox.component.scss

```diff
@@ -1,13 +1,36 @@
 @use 'variables' as *;
```

Comment:
Please use SCSS nesting inside this file a bit more to make the styling more readable